.ci/jenkins/dsl/jobs.groovy

```diff
@@ -142,36 +107,36 @@ void setupNativeJob() {
     }
 }
 
-void setupNativeLTSJob() {
-    def jobParams = getJobParams('optaplanner-native-lts', FolderUtils.getNightlyFolder(this), "${JENKINS_PATH}/Jenkinsfile.native", 'Optaplanner Native LTS Testing')
+void setupMandrelJob() {
```

Comment:
Does it mean that we are replacing the native LTS Quarkus check with Mandrel?

Reply:
Exactly. See also description on https://github.com/kiegroup/kogito-pipelines/pull/416

I will send a mail prior to the merge